### PR TITLE
Shows a hint when stub file is empty fixes #5

### LINKED ISSUE #5: Bug: Empty Component
I created a component with the generator. When I go to the component(which is blank yet) I get this error:
image

A nice solution would be to guide the user with the following steps.

### PR DESCRIPTION
**Pull request by jgnatch**
Shows a hint like this one when the stub file is empty:

image

### CODE REVIEW

File: app/views/mountain_view/styleguide/show.html.erb [hunk 1]
```diff
@@ -2,20 +2,33 @@
...
-<% @component.styleguide_stubs.each_with_index do |component_properties, index| %>
-  <div class="mv-component">
-    <div class="mv-component__item">
-      <%= render_component @component.name, component_properties %>
-    </div>
-    <div class="mv-component__description">
-      <h2><%= @component.title %> <%= index + 1 %></h2>
-      <div class="mv-component__description__definition">
-        <code class="language-ruby">&lt;%= render_component("<%= @component.name %>", {properties as below}) %&gt;</code>
+<% if @component.styleguide_stubs %>
```
Comment (kitop) [inline app/views/mountain_view/styleguide/show.html.erb]: What do you think of adding a `stubs?` method for this.

There could be a case where `styleguide_stubs` returns an empty array, and that would evaluate to true but not show any examples. We can make more complex checks in a new method, without polluting the view.

File: lib/mountain_view/component.rb [hunk 2]
```diff
@@ -14,8 +14,6 @@ def styleguide_stubs
       YAML.load_file stubs_file
     end
 
-    private
-
```
Comment (kitop) [inline lib/mountain_view/component.rb]: It'd be awesome to add a simple test to this method now that's not public :)

**Comment by kitop** [comment]
Love the idea and design!!

Left a few comments on the code. And one more thing:
What do you think of adding a simple integration test for this?

**Comment by jgnatch** [comment]
@kitop added tests!
Feel free to take a look again :grinning:

File: lib/mountain_view/component.rb [hunk 3]
```diff
@@ -14,10 +14,18 @@ def styleguide_stubs
...
     def stubs_file
       Rails.root.join("app/components/#{name}/#{name}.yml")
     end
+
+    def stubs?
+      if styleguide_stubs.is_a?(Array)
+        styleguide_stubs.any?
+      else
+        false
+      end
```
Comment (kitop) [inline lib/mountain_view/component.rb]: What do you think about doing a one-liner kind of thing? The `else false` part seems a bit redundant, both `is_a?` and `any?` already return the `true`/`false` we want.

Something like the following could work.
```
styleguide_stubs.is_a?(Array) && styleguides_stubs.any?
```

**Comment by kitop** [comment]
One small comment.
Nice changes! :+1:

**Comment by jgnatch** [comment]
@kitop Done! Much better now!

**Comment by kitop** [comment]
:shipit:

**Pull request merged**